src/node_report.cc

```diff
@@ -752,14 +853,15 @@ static void PrintStackFrame(FILE* fp, Isolate* isolate, Local<StackFrame> frame,
  * Function to print a native stack backtrace
  *
  ******************************************************************************/
-void PrintNativeStack(FILE* fp) {
+void PrintNativeStack(std::ostream& out) {
   void* frames[64];
-  fprintf(fp, "\n================================================================================");
-  fprintf(fp, "\n==== Native Stack Trace ========================================================\n\n");
+  out << "\n================================================================================";
+  out << "\n==== Native Stack Trace ========================================================\n\n";
 
   HANDLE hProcess = GetCurrentProcess();
   SymSetOptions(SYMOPT_LOAD_LINES | SYMOPT_UNDNAME | SYMOPT_DEFERRED_LOADS);
   SymInitialize(hProcess, nullptr, TRUE);
+  char buf[64];
```

Comment:
That was a bug, `buf` was being filled and the output was being truncated on Windows.
I've re-written the block so only the `"%2d: [pc=0x%p]"` part is formatted to `buf` as that's the only part that required the format specifiers, the rest can be written with `<<`.